### PR TITLE
Upgrade patternfly-react to v2.29.15

### PR DESCRIPTION
**Pull request by bond95**
In this version of patternfly-react appear SessionTimeout component (http://patternfly-react.surge.sh/patternfly-3/index.html?knob-displayBefore=30&knob-timeLeft=30&knob-primary%20content=Your%20session%20is%20about%20to%20expire&knob-secondary%20content=to%20continue%20click%20on%20%22Continue%20Session%22.&selectedKind=patternfly-react%2FCommunication%2FSession%20Timeout&selectedStory=Session%20Timeout&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs), which I need for fixing this issue https://github.com/oVirt/ovirt-web-ui/issues/955

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -81,7 +81,7 @@
     "moment-duration-format": "2.2.2",
     "patternfly": "3.53.1",
     "patternfly-bootstrap-combobox": "1.1.7",
-    "patternfly-react": "2.11.1",
+    "patternfly-react": "2.17.4",
```
Comment (sjd78) [inline package.json]: I think we should upgrade to the latest available as the risk to upgrade from 2.11.1 to 2.17.4 requires the same amount of regression testing (and potential fixes) as the upgrade from 2.11.1 to 2.29.15.

Also, which ever version gets selected, `patternfly` should be updated to the version used in `patternfly-react`.

Comment (gregsheremeta) [reply package.json]: agree

**Comment by sjd78** [comment]
@bond95 - Once the last tweaks are in, we can update `ovirt-engine-nodejs-modules` to pre-seed the changes and get standard-ci working again.  After standard-ci is working, then we can merge this one.

**Comment by sjd78** [comment]
With:
````
  devDeps:
    handlebars: 4.1.0
  deps:
    lodash: 4.17.11
    patternfly: 3.58.0
    patternfly-react: 2.29.15
````

I ran through the app on Chrome 72.0.3626.119 and everything looks to be in working order.  No obvious render or behavior issues.  :+1:

**Comment by bond95** [comment]
@sjd78 updated `patternfly` and added pre-seed https://gerrit.ovirt.org/#/c/98087/

**Comment by sjd78** [comment]
ci please build

**Pull request merged**